docs-kits/kits/Data Chain Kit/page_adoption-view.md

```diff
@@ -50,6 +51,19 @@ There are few steps for your Customer Journey:
 * The IRS Recursive API enables different use cases which collect insights over distributed data by not disclosing the supply chain
 * The solution enables interoperability with other solutions along the value chain to extend 
 
+<!-- !Mandatory! -->
+## Use Case
+
+### Environmental Social Responsibility (ESR)
+
+_Use Case for achieving compliance for Environmental Social Responsibility (ESR) for a component in the supply chain (multi-tier)_
+
+In the use case, it was assumed that a company in the supply chain is considered "compliant" if that company can show the ISO 14001 certificate. Presenting means that either the company itself provides it as a data provider or deposits it in a certificate repository with a service provider.
+The benefit of the solution is to obtain an automated overview of the supply chain's compliance and to be able to check it again and again on a cyclical basis. The companies in the supply chain are not disclosed, which follows the Catena-X specifications.
+The decentralized implementation at Catena-X "crawls" from the OEM to the direct supplier and from the direct supplier to its next supplier and thus through the supply chain for a specific component. Each partner determines its direct supplier automatically via the parts list stored by it in Catena-X's Item Relationship Service (IRS).
```

Comment:
I think "crawls" the verb crawling might results in neglecting thoughts of this solution. 
Maybe verbs like enable, allow, empower are more positive words.

Reply:
We should also add a picture to the working model of the Iterative so that people will understand this process better. 
Basically it is also built on the notification pattern of CX

Reply:
Is traverse a choice than crawl.

Reply:
Adding a picture will be a good idea. We should develop such a picture and once we have a scratch we need external support to align with the given KIT key visual designs. Shall we create an Enabler to create such a key visual within the next iterations?

Reply:
Tickets was created